go-apps/meep-loc-serv/server/loc-serv.go

```diff
@@ -311,6 +323,8 @@ func checkNotificationRegistrations(checkType int, param1 string, param2 string,
 
 func checkNotificationRegisteredZoneStatus(zoneId string, apId string, nbUsersInAPStr string, nbUsersInZoneStr string) {
 
+	mutex.Lock()
```

Comment:
Recommend using the following syntax here:
mutex.Lock()
defer mutex.Unlock()

This helps avoid cases where a return statement is added without unlocking the mutex. For longer functions it makes it simpler to implement. I recommend this approach for the larger functions to prevent future bugs.

Reply:
defer added everywhere to prevent future error.